backend/s3/s3.go

```diff
@@ -2878,6 +2879,8 @@ type Options struct {
 	UseUnsignedPayload    fs.Tristate          `config:"use_unsigned_payload"`
 	SDKLogMode            sdkLogMode           `config:"sdk_log_mode"`
 	DirectoryBucket       bool                 `config:"directory_bucket"`
+	APIKey                string               `config:"api_key"`
```

Comment:
There don't appear to be matching config definitions for `APIKey` or `InstanceID`. Which also means they have no help.

Since these are ibm specific they should probably be prefixed `ibmcos_api_key` and `ibmcos_resource_instance_id`.

Reply:
addressed